app/src/dev/java/cm/aptoide/pt/app/view/AppViewPresenter.java

```diff
@@ -841,16 +810,15 @@ private void continueLoggedInRecommendsDialogClick() {
         .filter(lifecycleEvent -> lifecycleEvent == View.LifecycleEvent.CREATE)
         .flatMap(created -> view.continueLoggedInRecommendsDialogClick()
```

Comment:
If this refers to the continue option of the logged in recommends dialog, then this should be renamed because there is no more continue option